src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -217,6 +219,24 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       }
     }
   }
+  /* Return a Markdown table of all entries in the database */
```

Comment:
I think that this should be somehow toggleable - I don't think it should be written out all over the place unless someone wants to see it